rockhound/mcmurray_facies.py

```diff
@@ -0,0 +1,100 @@
+    Wabiskaw Member of the Clearwater Formation in the Athabasca Oil Sands Area. The data that
+    accompany this report are one of the most significant products of the project and will hopefully
+     facilitate future development of the oil sands.” It includes well log curves as LAS
+     files and tops in txt files and xls files. There is a word doc and a text file that
+     describes the files and associated metadata.
+
+    _Wynne, D.A., Attalla, M., Berezniuk, T., Brulotte, M., Cotterill, D.K., Strobl, R.
+    and Wightman, D. (1995): Athabasca Oil Sands data McMurray/Wabiskaw oil sands deposit
+     - electronic data; Alberta Research Council, ARC/AGS Special Report 6._
+
+    Please go to the links below for more information and the dataset:
+
+    Report for Athabasca Oil Sands Data McMurray/Wabiskaw Oil Sands Deposit
+    http://ags.aer.ca/document/OFR/OFR_1994_14.PDF
+
+    Electronic data for Athabasca Oil Sands Data McMurray/Wabiskaw Oil Sands Deposit
+    http://ags.aer.ca/publications/SPE_006.html
+
+    In the metadata file
+    https://github.com/JustinGOSSES/MannvilleGroup_Strat_Hackathon/blob/master/
+    SPE_006_originalData/Metadata/SPE_006.txt -> SPE_006.txt.
+    the dataset is described as Access Constraints: Public and Use Constraints:
+    Credit to originator/source required. Commercial reproduction not allowed.
+
+    _The Latitude and longitude of the wells is not in the original dataset.
+    https://github.com/dalide -> @dalide used the Alberta Geological Society’s UWI
+    conversion tool to find lat/longs for each of the well UWIs. A CSV with the
+    coordinates of each well’s location can be found
+    https://github.com/JustinGOSSES/MannvilleGroup_Strat_Hackathon/blob/master/well_lat_lng.csv.
+    These were then used to find each well’s nearest neighbors.
+
+    Please note that there are a few misformed .LAS files in the full dataset, so the code
+    in this repository skips those.
+
+    If for some reason the well data is not found at the links above, you should
+    be able to find it
+    https://github.com/JustinGOSSES/MannvilleGroup_Strat_Hackathon/tree/master/SPE_006_originalData
+
+    If the file isn't already in your data directory, it will be downloaded
+    automatically.
+
+    Parameters
+    ----------
+    load : bool
+        Wether to load the data into a :class:`pandas.DataFrame` or just return the
+        path to the downloaded data.
+
+    Returns
+    -------
+    mcmurray_facies : :class:`pandas.DataFrame` or str
+        The loaded data or the file path to the downloaded data.
+        The :class:`pandas.DataFrame` contains the following data:
+
+        columns : ['CALI', 'COND', 'DELT', 'DEPT', 'DPHI', 'DT', 'GR', 'ILD', 'ILM',
+        'NPHI', 'PHID', 'RHOB', 'SFL', 'SFLU', 'SN', 'SP', 'UWI', 'SitID','lat', 'lng',
+         'Depth', 'LithID', 'W_Tar', 'SW', 'VSH', 'PHI', 'RW','lithName']
```

Comment:
I'm not familiar with type of dataset but these names mean nothing to me. What would you think of renaming them to something more verbose? For example, I try to use `latitude` and `longitude` instead of `lat` `lng`/`lon`/`long`. With autocomplete in the notebooks, the length of the names doesn't matter much when typing them out. This can be done when loading the data here so you don't have to change the original files.

Reply:
Thanks for all the feedback. 

With exception of 'lat' and 'lng', the abbreviations are standards. I worry about switching to spelled out versions as some people who use this type of data regularly wouldn't then know what curve it was. They only know the abbreviation. 

How about supplying a separate dictionary? Or renaming to long form as default behavior but allowing for abbreviations via an argument submitted to rh.fetch_mcmurray_facies(curveNameAbrev=true)?

Reply:
Oh OK, if it's standard in the field then there is no need to rename. I would recommend including the translation for each abbreviation in the docstring, though. Some people might be looking to get into the field and not know all of the acronyms by heart yet.

Reply:
Went the abbreviations as default column names but an option to get abbreviation+spelled out column names.

To get the spelled out column names, you're required to add an argument like this: `fetch_mcmurray_facies(abbreviations_only=False)`.  

Default behavior with no arguments `fetch_mcmurray_facies()` is the same thing as calling `fetch_mcmurray_facies(abbreviations_only=True, load=True)`. 

Also included both versions in docstrings.